src/alfrescoApi.ts

```diff
@@ -544,21 +544,21 @@ export class AlfrescoApi implements Emitter {
      * Get the current Ticket for the Bpm
      * */
     getTicketBpm(): string {
-        return this.processAuth && this.processAuth.getTicket();
+        return this.isOauthConfiguration() ? this.config.ticketBpm : this.oauth2Auth.getToken();
```

Comment:
shouldn't it be the opposite ? 
if is OAUTH return this.oauth2Auth.getToken else this.config.ticketBpm  ?